packages/checkout/widgets-lib/src/lib/hooks/useAnimation.ts

```diff
@@ -0,0 +1,24 @@
+export const useAnimation = () => {
```

Comment:
I think the you don't require a hook for keeping this static values, it will more efficient to directly import them. Thoughts?

Reply:
Yep totally agree! will move it out

Reply:
Fixed.